src/Unleash/Internal/FeatureEvaluationResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using Unleash.Variants;
+
+namespace Unleash.Internal
+{
+	public class FeatureEvaluationResult
```

Comment:
urgh, I'll fix this

Reply:
Is the fix that this should be private? I think this should be private

Reply:
it was going to be internal, I think private isn't going to work.. the fix however was aimed at the indentation tabs->spaces issue

Reply:
Whatever the .NET version of "user-no-touchy" is